packages/musl/1.2.2/chksum

```diff
@@ -0,0 +1,4 @@
+aed8ae9e2b0898151b36a204088292dd  musl-1.2.2.tar.gz
```

Comment:
These should be in the format <algo> <filename> <hash>

i.e.
`md5 musl-1.2.2.tar.gz aed8ae9e2b0898151b36a204088292dd`